tutorials/VI_in_BM.ipynb

```diff
@@ -0,0 +1,2009 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        return dist.Normal(phi_mean, phi_sd.softplus())
</pre><p> </p><p>wait how did you get this to run? I thought torch tensor/ parameters doesn't have a <code>.softplus()</code> attribute? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Oh no, I think well I rearranged cells some got deleted. Let me fix this!